src/proj.h

```diff
@@ -410,6 +427,13 @@ PJ_PROJ_INFO proj_pj_info(PJ *P);
 PJ_GRID_INFO proj_grid_info(const char *gridname);
 PJ_INIT_INFO proj_init_info(const char *initname);
 
+/* List functions: */
+/* Get lists of operations, ellipsoids, datums, units and prime meridians. */
+PJ_OPERATIONS       *proj_list_operations(void);
```

Comment:
I'd suggest to prefix the return types with const so that the user isn't tempted to modify them

Reply:
Good point. I'll fix it right away.